pennylane/ops/op_math/controlled.py

```diff
@@ -73,10 +73,8 @@ def circuit(x):
     :func:`~.ctrl` works on both callables like ``qml.RX`` or a quantum function
     and individual :class:`~.operation.Operator`'s.
 
-    >>> qml.ctrl(qml.PauliX(0), (1,2))
-    Controlled(PauliX(wires=[0]), control_wires=[1, 2])
-    >>> qml.ctrl(qml.PauliX(0), (1,2)).decomposition()
-    [Toffoli(wires=[1, 2, 0])]
```

Comment:
Do we want to preserve the decomposition example with the controlled Hadamard?

Reply:
it seems irrelevant to the description above tbh, which is why I dropped it. plus, only `PauliX` had nice decompositions lol